docker/l4t/32.1/nano/Dockerfile

```diff
@@ -1,10 +1,21 @@
 ARG VERSION_ID
+# using a smaller image doesn't save us any space as the layers are used by the other images.
+FROM ubuntu:${VERSION_ID} as qemu
+
+# install qemu for the support of building containers on host
+RUN apt-get update && apt-get install -y --no-install-recommends qemu-user-static && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
 FROM arm64v8/ubuntu:${VERSION_ID}
 
 ARG URL=https://developer.download.nvidia.com/embedded/L4T/r32_Release_v1.0/jetson-nano/BSP
 
 ARG DRIVER_PACK=Jetson-Nano-Tegra210_Linux_R32.1.0_aarch64.tbz2
 
+# pre-req qemu binary for cross-compilation on x86
```

Comment:
See previous comment from jax-tx2/Dockerfile

Reply:
Let me know if this reads better:
# pre-req qemu binary for aarch64 emulation if this container image is built directly on x86 host